tokens/dist/index.css

```diff
@@ -3688,9 +3688,8 @@
 
   --spectrum-tooltip-animation-distance:var(--spectrum-spacing-75);
 
-  /* To get a square that measure 10px on the diagonal, the sides have to be 7px, but this has to
-		 be 8px because we have to account for antialiasing so there isn't a gap between tooltip and tip */
-	--spectrum-tooltip-tip-square-size:8px;
+  /* To get a square that measures 10px on the diagonal, the sides have to be 7px */
+	--spectrum-tooltip-tip-square-size:7px;
```

Comment:
This small discrepancy in size was called out in the design review--clearly there was a reason `8px` was used instead of `7px`, and `10px` rather than `9px` for large, but testing it again I'm not seeing it, so I'm having a hard time justifying it, I'd love a second opinion here.

Reply:
I don't think I understand enough about anti-aliasing. The only way I can see that hairline is if I change `--spectrum-tooltip-tip-antialiasing-inset` to 0px. 

I do not see any gap between the tip and the rest of the tooltip with this changed to 7px, or to the 9px (as long as we keep that property I mentioned at 0.5px~)